src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiOutPort.ts

```diff
@@ -0,0 +1,21 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiOutPort {
+		public static sendDefault(encodedDeviceId: string) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			var deviceId = decodeURIComponent(encodedDeviceId);
+			var noteOnMessage = [0x90, 60, 0x7f];    // note on, middle C, full velocity
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'noteOnMessage' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)